src/utils/slpReader.js

```diff
@@ -481,3 +484,22 @@ export function getMetadata(slpFile: SlpFileType): MetadataType {
   // $FlowFixMe
   return metadata;
 }
+
+export function getGameEnd(slpFile: SlpFileType): GameEndType {
+  const gameEndLength = slpFile.messageSizes[0x39] + 1;
+  if (gameEndLength <= 1 || slpFile.metadataLength <= 0) {
+    // This will happen on a severed incomplete file
+    // $FlowFixMe
+    return {};
+  }
+
+  const buffer = new Uint8Array(gameEndLength);
+  const start = slpFile.metadataPosition - METADATA_OFFSET - gameEndLength;
+  readRef(slpFile.ref, buffer, 0, gameEndLength, start);
+  const view = new DataView(buffer.buffer);
```

Comment:
I might prefer to reuse existing code here? Might be especially useful if the game end payload ever changes sizes.

Something like:
```
const command = buffer[0];
const payload = buffer.slice(1);
return parseMessage(command, payload);
```